src/Compiler/Service/ServiceParseTreeWalk.fs

```diff
@@ -819,7 +819,7 @@ module SyntaxTraversal =
                 | SynType.Array (_, ty, _) -> traverseSynType path ty
                 | SynType.StaticConstantNamed (ty1, ty2, _)
                 | SynType.MeasureDivide (ty1, ty2, _) -> [ ty1; ty2 ] |> List.tryPick (traverseSynType path)
-                | SynType.Tuple (_, tys, _) -> tys |> List.map snd |> List.tryPick (traverseSynType path)
+                | SynType.Tuple(_, _, _, elements, _) -> elements |> List.map snd |> List.tryPick (traverseSynType path)
```

Comment:
@edgarfgp is it important that we still include the `firstType` into the existing code.
I think here we need something like `firstType :: (List.map snd elements) |> List.tryPick (traverseSynType path)
Write Preview`.

Reply:
I had some time today and I made a change based on your suggestion :)